.solcover.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
     skipFiles: [
       'Migrations.sol',
       'EtherRouter.sol',
-      'PatriciaTree'
+      'PatriciaTree',
+      'ContractEditing' // Covered implicitly via the colony-network-recovery.js tests
```

Comment:
I agree it doesn't need to factor in to our coverage, but I don't understand the comment? I'd say it was because it's only used in setting up tests, never in production.